docs/install-and-run.md

```diff
@@ -285,3 +285,40 @@ compatible with EarlGrey.
 can run it by just clicking on the **run** icon or by Ctrl+clicking the test name and then selecting
 **Test "testPresenceOfKeyWindow"**. Because this is a regular unit test, you can place breakpoints in
 your test and in your application code and use the built-in tools seamlessly.
+
+### Ruby Installation
```

Comment:
Thanks for adding this. You should link to to one of the install steps above so user can jump to this section before they install EarlGrey gem, etc.